### PR TITLE
Added instructions to request compute node for conda environment creation

### PR DESCRIPTION
**Pull request by suryap649**
Updated the User Facing Documentation with instructions to request compute node for conda environment creation. Builded and tested the project. More details regarding this PR can be found here - [INC8326525](https://service.northeastern.edu/incident.do?sys_id=ee8d78a247b49550b9dd586c346d4365&sysparm_record_target=incident&sysparm_record_row=3&sysparm_record_rows=4&sysparm_record_list=assignment_group%3Ddec535864f7abe4026868f85f110c72f%5EstateIN6%2C7%5Eactive%3Dfalse%5Eclosed_atONThis+month%40javascript%3Ags.beginningOfThisMonth%28%29%40javascript%3Ags.endOfThisMonth%28%29%5Eassigned_to%3D8e67451d47d785d0a75b2172e36d4354%5EORDERBYzztextsearchyy)

### CODE REVIEW

**Review by manasvita — APPROVED** [review]
This looks good to me.

**Comment by manasvita** [comment]
@suryap649, checks are still failing. Merging may not happen until that's taken care of.

**Comment by ShobanaSekar** [comment]
@suryap649 For the checks failing issue, you need to update 
sphinx-markdown-tables==0.0.16
inside docs/requirements.txt

**Comment by suryap649** [comment]
@ShobanaSekar @manasvita  @marianalevineu , I have updated the sphnix-markdown-tables version in the requirements.txt file. The builds are successful now.

File: docs/source/software/conda.rst [hunk 1]
```diff
@@ -15,14 +15,15 @@ Creating a Conda virtual environment with Anaconda
...
-2. To load anaconda, type ``module load anaconda3/2022.01``.
-3. To create your environment, type ``conda create -n <yourenvironmentname> python=3.7 anaconda``, where <yourenvironmentname> is the name you want to give your environment. Tip: to see a list of all of your conda environments, type ``conda info -e``. To save space, also use the ``--prefix=/work/<mygroup>/<mydirectory>`` flag to build in your work directory.
-4. Follow the prompts to complete the Conda install.
-5. To activate your Conda environment, type ``source activate <yourenvironmentname>``. Note that ``conda activate`` will not work on Discovery with this version.
-6. To install a specific package, type ``conda install -n <yourenvironmentname> [package]``.
-7. To deactivate the current, active Conda environment, type ``conda deactivate``.
-8. To delete a Conda environment and all of its related packages, type ``conda remove -n <yourenvironmentname> --all``.
+1. If you are on a login node, move to a compute node by typing ``srun --partition=short --nodes=1 --cpus-per-task=1 --pty /bin/bash``. Allocate resources as per your requirements.
```
Comment (ShobanaSekar) [inline docs/source/software/conda.rst:18]: Before starting with "allocate resources...", you can say something like:
"In the above example, we request for 1 node and 1 cpu, but you can request for additional resources as per your requirements.

**Review by ShobanaSekar — CHANGES_REQUESTED** [review]

File: docs/source/software/conda.rst (hunk 1 above)
Comment (suryap649) [reply docs/source/software/conda.rst]: @ShobanaSekar  I have made the changes you mentioned and have tested it out.

**Review by ShobanaSekar — APPROVED** [review]

**Comment by marianalevineu** [comment]
thanks for working on it @suryap649 looks good!

**Pull request merged**